tuned-main.conf

```diff
@@ -75,3 +75,6 @@ log_file_max_size = 1MB
 # Size of connections backlog for listen function on socket
 # Higher value allows to process requests from more clients
 # connections_backlog = 1024
+
+# Perform rollback on TuneD daemon exit
+# rollback_on_exit = 1
```

Comment:
Maybe for future extensibility use `rollback` option name and string values e.g. `on_exit` and `auto` with the default value `auto`, e.g.:
```
rollback = on_exit
```
And we could add later more options like e.g.: `never`, `always`, etc.

We should also add the `tuned-main.conf` description to the upstream documentation, but this could be done later. It seems there is no dedicated section in the documentation yet.

Reply:
Thank you for the suggestion.  I believe we need "auto" and "not_on_exit" right now.  Code adjusted.